hoomd/md/pair/pair.py

```diff
@@ -293,6 +293,63 @@ def __init__(self, nlist, default_r_cut=None, default_r_on=0., mode='none'):
         super().__init__(nlist, default_r_cut, default_r_on, mode)
 
 
+class ExpandedGaussian(Pair):
+    r"""Expanded Gaussian pair force.
+
+    Args:
+        nlist (hoomd.md.nlist.NeighborList): Neighbor list.
+        default_r_cut (float): Default cutoff radius :math:`[\mathrm{length}]`.
+        default_r_on (float): Default turn-on radius :math:`[\mathrm{length}]`.
+        mode (str): Energy shifting/smoothing mode.
+
+    `ExpandedGaussian` computes the Gaussian pair force should on every particle
+    in the simulation state:
+
+    .. math::
+        U(r) = \varepsilon \exp \left( -\frac{1}{2}
+               \left( \frac{r-\delta}{\sigma} \right)^2 \right)
+
+    Example::
+
+        nl = nlist.Cell()
+        sgauss = pair.ExpandedGaussian(default_r_cut=3.0, nlist=nl)
+        sgauss.params[('A', 'A')] = dict(epsilon=1.0, sigma=1.0, delta=0.5)
+        sgauss.r_cut[('A', 'B')] = 3.0
+
+    .. py:attribute:: params
+
+        The expanded Gauss potential parameters. The dictionary has the
```

Comment:
```suggestion
        The expanded Gaussian potential parameters. The dictionary has the
```

Reply:
I updated the `phinx-doc/module-md-pair.rst` and the related doc part in the `pair/pair.py`.